### PR TITLE
One build to rule them all

### PR DESCRIPTION
**Pull request by dougbu**
- well, at least VS and command-line builds will share output
- part of aspnet/Coherence-Signed#277

### CODE REVIEW

**Comment by dougbu** [comment]
Once this is signed off, I'll run the same script against all other repos in quick succession. Not planning on reviewing any of those unless I hit odd cases.

File: samples/MvcSandbox/MvcSandbox.xproj [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <DnxInvisibleContent Include="bower.json" />
     <DnxInvisibleContent Include=".bowerrc" />
     <DnxInvisibleContent Include="package.json" />
+    <DnxInvisibleContent Include=".npmrc" />
   </ItemGroup>
   <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.targets" Condition="'$(VSToolsPath)' != ''" />
```
Comment (BrennanConroy) [inline samples/MvcSandbox/MvcSandbox.xproj]: Shouldn't the Imports section be changed to DOTNET?

File: samples/MvcSandbox/MvcSandbox.xproj [hunk 2]
```diff
@@ -17,6 +17,7 @@
     <DnxInvisibleContent Include="bower.json" />
     <DnxInvisibleContent Include=".bowerrc" />
     <DnxInvisibleContent Include="package.json" />
+    <DnxInvisibleContent Include=".npmrc" />
```
Comment (BrennanConroy) [inline samples/MvcSandbox/MvcSandbox.xproj]: Is this still DnxInvisibleContent?

File: samples/MvcSandbox/MvcSandbox.xproj (hunk 1 above)
Comment (dougbu) [inline samples/MvcSandbox/MvcSandbox.xproj]: Discussed w/ @BillHiebert and we decided that's not worthwhile. For one thing, my script would need to duplicate the logic in `Microsoft.DNX.targets` to choose between `Microsoft.DotNet.Targets` and `Microsoft.DotNet.Web.Targets`. For another, we should eventually switch to `.csproj` files anyway.

File: samples/MvcSandbox/MvcSandbox.xproj (hunk 2 above)
Comment (dougbu) [inline samples/MvcSandbox/MvcSandbox.xproj]: Yup, checked a recently-created project.

Comment (dougbu) [inline samples/MvcSandbox/MvcSandbox.xproj]: BTW I added the additional "ignore me" line because newer projects have it.

**Comment by pranavkm** [comment]
Does this affect the test runner in any way? I've seen it specify the artifacts dir when it runs `dotnet test`. Hopefully that's not hardcoded.

**Comment by pranavkm** [comment]
:shipit:

**Comment by BrennanConroy** [comment]
:shipit:

**Comment by dougbu** [comment]
> Does this affect the test runner in any way?

If a command-line build goes first, test discovery can start when opening one of the MVC solution files. Doesn't wait for a VS build anymore.

> I've seen it specify the artifacts dir when it runs `dotnet test`. Hopefully that's not hardcoded.

Not hardcoded. The `dotnet test` commands I see all have updated `--output ...` arguments.

**Comment by dougbu** [comment]
7430efa

**Pull request merged**